webviz_config/command_line.py

```diff
@@ -175,3 +175,14 @@ def entrypoint_schema(args: argparse.Namespace) -> None:
     args = parser.parse_args()
 
     args.func(args)
+
+
+def _dummy_create_ca() -> None:
```

Comment:
The set_defaults sends the arguments to the function, currently fails.
```suggestion
def _dummy_create_ca(_: argparse.Namespace) -> None:
```